index.html

```diff
@@ -6168,29 +6203,30 @@ <h3 id="example-2">
           </dt>
           <dd>
             The frequency at which the <a><code>BiquadFilterNode</code></a>
-            will operate, in Hz. Its default value is 350Hz, and its nominal
-            range is from 10Hz to half the Nyquist frequency.
+            will operate, in Hz. Its default value is 350Hz. It forms a
+            <a>compound parameter</a> with <code>detune</code>.
           </dd>
           <dt>
             readonly attribute AudioParam detune
           </dt>
           <dd>
             A detune value, in cents, for the frequency. Its default value is
-            0.
+            0. It forms a <a>compound parameter</a> with <code>frequency</code>.
           </dd>
           <dt>
             readonly attribute AudioParam Q
           </dt>
           <dd>
             The <a href="https://en.wikipedia.org/wiki/Q_factor">Q</a> factor
-            has a default value of 1, with a nominal range of 0.0001 to 1000.
+            has a default value of 1. Its <a>nominal range</a> is [0.0001,
+            1000].
```

Comment:
Do we really need to limit the Q values to this range? From the Wikipedia article, I think we only need to limit Q to the range (0, inf).